brouter-routing-app/src/main/java/btools/routingapp/BRouterActivity.java

```diff
@@ -76,6 +87,24 @@ public void onCreate( Bundle savedInstanceState )
     // Create a bright wake lock
     mWakeLock = mPowerManager.newWakeLock( PowerManager.SCREEN_BRIGHT_WAKE_LOCK, getClass().getName() );
 
+    // CHANGE: consider start with Intent to import a profile
+    Intent intent = getIntent();
+    String action = intent.getAction();
+    String type = intent.getType();
+    System.out.println("Brouter START xxxxxx BrouterActivity");
+    if (Intent.ACTION_VIEW.equals(action) ) {
+      System.out.println("Brouter Intent.ACTION_VIEW detected");
+      import_profile(intent);
+      return;
+    }
+// end change
+
+    // Get an instance of the PowerManager
+    //mPowerManager = (PowerManager) getSystemService( POWER_SERVICE );
+
+    // Create a bright wake lock
+    //mWakeLock = mPowerManager.newWakeLock( PowerManager.SCREEN_BRIGHT_WAKE_LOCK, getClass().getName() );
+
```

Comment:
Unrelated changes

Reply:
Clear this comments (I had to create the mWakeLock before calling the import_profile, else an error occured on the onresume,,,)